www/js/diary/services.js

```diff
@@ -1,15 +1,13 @@
 'use strict';
 
 import angular from 'angular';
-import { getBaseModeByKey, getBaseModeOfLabeledTrip } from './diaryHelper';
 import { SurveyOptions } from '../survey/survey';
 import { getConfig } from '../config/dynamicConfig';
-import { getRawEntries } from '../commHelper';
 
 angular.module('emission.main.diary.services', ['emission.plugin.logger',
```

Comment:
I am running into some issues when I remove this module.  I've deleted all references to the `Timeline` factory (e.g., the one in `LabelTab.tsx`), and have been able to remove it without any issues.  From what I can tell, there are only two remaining references to this module:
- the `emission.main.diary` service uses this module as a provider
- `index.js` imports the entire file

If I remove either of these references, the timeline breaks and freezes on a blank screen, without throwing an error or warning.  This made me thing one of these imports is lacking a provider higher up on the dependency chain - however, if I remove all of `emission.main.diary.services` providers, everything functions normally, despite the fact that the module now contributes no new code.  I figure I've missed something important, I'll update this thread with notes as needed!